apps/meteor/app/cloud/server/methods.ts

```diff
@@ -151,7 +150,9 @@ Meteor.methods<ServerMethods>({
 			});
 		}
 
-		return disconnectWorkspace();
+		await disconnectWorkspace();
```

Comment:
instead of calling a function that has a meaningless error I'd suggest to return an error/message that actually tells to the client/end user what is going on.

Reply:
I guess we need to also deprecate `cloud:disconnectWorkspace` method, right?